src/BinaryParsers/ElfBinary/Dwarf/DwarfCompilationUnit.cs

```diff
@@ -515,6 +539,7 @@
                     {
                         DwarfAttribute attribute = (DwarfAttribute)debugDataDescription.LEB128();
                         DwarfFormat format = (DwarfFormat)debugDataDescription.LEB128();
+                        object value = null;
 
                         while (format == DwarfFormat.Indirect)
```

Comment:
>ImplicitConst [](http://example.com/codeflow?start=50&length=13)

This is the code path that caused our break. The failure to retrieve this value and move the stream position pointer forward caused subsequent corruption in parsing.